pkg/unpack/unpacker.go

```diff
@@ -162,7 +163,7 @@ func (u *Unpacker) Unpack(h images.Handler) images.Handler {
 		layers = map[digest.Digest][]ocispec.Descriptor{}
 	)
 	return images.HandlerFunc(func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
-		ctx, span := tracing.StartSpan(ctx, "pkg.unpack.unpacker.UnpackHandler")
+		ctx, span := tracing.StartSpan(ctx, tracing.SpanName(unpackSpanPrefix, "UnpackHandler"))
```

Comment:
Do we need both `StartSpan` and `SpanName` ?

I'd suggest
```go
ctx, span := tracing.StartSpan(ctx, unpackSpanPrefix, "UnpackHandler")
```

Reply:
`tracing.StartSpan`   already accepts `SpanStartOption...` 
```
func StartSpan(ctx context.Context, opName string, opts ...trace.SpanStartOption) (context.Context, *Span)
```
so I had to use `tracing.SpanName` to pass the list of strings to create the span name.  

The other options were to  use `interface{}` and infer the type later,  or create a new method that accepts `StartSpan(ctx, name ...string)` but then it cannot take `SpanStartOption...`

Reply:
Oh, ok, that make sense.
In this case I'd suggest `tracing.Name` instead of `tracing.SpanName` to keep it more laconic.